.github/workflows/deploy-vercel-staging.yml

```diff
@@ -118,10 +118,25 @@ jobs:
           env: ${{ env.CUSTOMER_REF_TO_DEPLOY }}-${{ env.STAGE }} # Uses "$customer-$env" as GitHub environment name, because it's easier this way to see what has changed per customer, per stage
           no_override: true # Disables auto marking previous environments as "inactive", as they're still active (Vercel deployments don't auto-deactivate) and it would remove the previous deployment links needlessly
 
+      # extract branch name on branch push
+      # Limit the branch name length to keep a readable URL
+      - name: Extract branch name (in case of push on branch)
+        if: github.event_name != 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | cut -c1-15)" >> $GITHUB_ENV
```

Comment:
How are we supposed to know it should be 15 chars?

Shouldn't that rather be handled by the action?